src/audio-player-composable/audio-player-composable.tsx

```diff
@@ -100,26 +100,23 @@ export const AudioPlayerComposable = ({
     };
   };
 
+  const getSeekBarProps = () => ({
+    duration,
+    trackPositionArr,
+    onChangeSlider,
```

Comment:
if we can rename that so its user friendly
```suggestion
    onChange: onChangeSlider,
```